capstone/src/test/java/com/google/univiz/api/SearchResourceImplTest.java

```diff
@@ -0,0 +1,76 @@
+package com.google.univiz.api;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.mockito.Mockito.when;
+
+import com.google.common.collect.Lists;
+import com.google.univiz.api.representation.CollegeId;
+import com.google.univiz.api.representation.SearchData;
+import com.google.univiz.api.representation.SuggestionData;
+import com.google.univiz.api.representation.SuggestionResponse;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+
+@RunWith(JUnit4.class)
+public final class SearchResourceImplTest {
+  @Mock private SuggestionDataApi mockSuggestionApi;
+  @Mock private SuggestionResponse cannedResponse;
```

Comment:
nit - since it's pretty lightweight to construct these objects, these tests would be more readable if you used some constant data instead of mocks. you can set them up once as constants in this class and that way every test method doesn't need to configure these mocks.